tests/Tokenizer/Analyzer/NamespaceUsesAnalyzerTest.php

```diff
@@ -139,12 +140,91 @@ public function provideNamespaceUsesCases()
-            ], [1]],
+            ], true],
 
-            // TODO: How to support these:
+            ['<?php use Foo\Bar, Foo\Baz;', [], true],
+            ['<?php use Foo\Bar, Foo\Baz;', [
+                new NamespaceUseAnalysis(
+                    'Foo\Bar',
+                    'Bar',
+                    false,
+                    1,
+                    6,
+                    NamespaceUseAnalysis::TYPE_CLASS
+                ),
+                new NamespaceUseAnalysis(
+                    'Foo\Baz',
+                    'Baz',
+                    false,
+                    7,
+                    11,
+                    NamespaceUseAnalysis::TYPE_CLASS
+                ),
+            ], false],
+            ['<?php use Foo\Bar as Baz, Some\ClassA as Alias,No\Space;', [
+                new NamespaceUseAnalysis(
+                    'Foo\Bar',
+                    'Baz',
+                    true,
+                    1,
+                    10,
+                    NamespaceUseAnalysis::TYPE_CLASS
+                ),
+                new NamespaceUseAnalysis(
+                    'Some\ClassA',
+                    'Alias',
+                    true,
+                    11,
+                    19,
+                    NamespaceUseAnalysis::TYPE_CLASS
+                ),
+                new NamespaceUseAnalysis(
+                    'No\Space',
+                    'Space',
+                    false,
+                    20,
+                    23,
+                    NamespaceUseAnalysis::TYPE_CLASS
+                ),
+            ], false],
+            ['<?php use function some\a\fn_a as alias, some\a\fn_b as b;', [
+                new NamespaceUseAnalysis(
+                    'some\a\fn_a',
+                    'alias',
+                    true,
+                    1,
+                    14,
+                    NamespaceUseAnalysis::TYPE_FUNCTION
+                ),
+                new NamespaceUseAnalysis(
+                    'some\a\fn_b',
+                    'b',
+                    true,
+                    15,
+                    25,
+                    NamespaceUseAnalysis::TYPE_FUNCTION
+                ),
+            ], false],
+            ['<?php use const some\a\ConstA as A, ConstB;', [
+                new NamespaceUseAnalysis(
+                    'some\a\ConstA',
+                    'A',
+                    true,
+                    1,
+                    14,
+                    NamespaceUseAnalysis::TYPE_CONSTANT
+                ),
+                new NamespaceUseAnalysis(
+                    'ConstB',
+                    'ConstB',
+                    false,
+                    15,
+                    17,
+                    NamespaceUseAnalysis::TYPE_CONSTANT
+                ),
+            ], false],
 
-            // Multiple imports on one line:
-            // use My\Full\Classname as Another, My\Full\NSname;
+            // TODO: How to support these:
```

Comment:
i guess this is not yet resolved?

Reply:
not resolved, NamespaceUseAnalysis wont fit for group import
start and end tokens do not apply as the use is not in one piece